homeassistant/components/iss/config_flow.py

```diff
@@ -34,7 +36,11 @@ async def async_step_user(self, user_input=None) -> FlowResult:
 
         if user_input is not None:
             return self.async_create_entry(
-                title="International Space Station", data=user_input
+                title="International Space Station",
+                data={CONF_NAME: user_input.get(CONF_NAME, DEFAULT_NAME)}
+                if CONF_NAME in user_input
+                else user_input,
```

Comment:
Where is this data used? Shouldn't this be used for the integration title instead?

Reply:
It is for importing the `name` of the entity people have defined in their now imported config. This will be removed in 2022.5 as it is not need after we remove the import from `yaml`

Reply:
But if we remove it at that time, it will still be a breaking change?
So I'm not sure why it is passed into the entry data, later used; to remove later. While new configuration entries can't set it?

Reply:
It is only used if it is imported. New users have no use for it, as they can change the name in the UI.

Reply:
> But what would a better way be to not make it a breaking change in 3 months?

Setting it as the config entry title, and use that for the device name as well?

Reply:
I see that I am a noob at this. 😭😆 You are right, I thought the title was for something else. Thanks!

Reply:
Thanks for the help, even though I didn't understand what you meant at first 🙏